package.json

```diff
@@ -21,9 +21,9 @@
   ],
   "scripts": {
     "build": "tsc",
-    "dev": "ts-node ./src/app.ts",
+    "dev": "ts-node ./src/testScripts/sdkV1-getters.ts",
```

Comment:
Could be nice to add a small README about why/what running the test script does

Reply:
it runs the content of the test script which goes through a bunch of getters in sdkV1

i'll add a ticket to add a README https://linear.app/cega/issue/CEG-1031/add-readme-about-scheduler-packagejson-scripts-to-be-run